.github/workflows/samples-dotnet6-client.yaml

```diff
@@ -37,9 +37,9 @@ on:
       - samples/client/petstore/csharp/OpenAPIClient-net47/**
       # build C# API client (.net framework 4.8)
       - samples/client/petstore/csharp/OpenAPIClient-net48/**
-      # build C# API client (.net 5.0)
-      - samples/client/petstore/csharp/OpenAPIClient-net5.0/**
-      # build C# API client (.net 5.0 with ConditionalSerialization)
+      # build C# API client (.net 7.0)
+      - samples/client/petstore/csharp/OpenAPIClient-net7.0/**
```

Comment:
In each workflow, we only need to test the samples which target that framework, so this file only needs .net 6 samples.

Reply:
right, will do